### PR TITLE
Feature/idcom 2197  readme update

### PR DESCRIPTION
**Pull request by luke-song**

### CODE REVIEW

**Review by JulianSpring — APPROVED** [review]

File: solana/program_v2/packages/client/core/README.md [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+
+A typescript client library for interacting with Identity.com's On-Chain Identity Gateway
+
+## Features
+
+The `gateway-solana-client` library provides the following features:
+
+1. AdminService provides create, update, close, and retrieve Network account methods.
+2. NetworkService provides create, update, close, retreive and setState of Gatekeeper methods.
+3. gatekeeperService provides issue, refresh, setState, setData, change the Associated Gatekeeper, expire, and verify of pass account methods.
```
Comment (DiosAkhilleus) [inline solana/program_v2/packages/client/core/README.md:11]: Gatekeeper on GatekeeperService is capitalized

File: solana/program_v2/packages/client/core/README.md [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+4. Typescript Client library for AdminService, NetworkService, and GatewayService.
+5. It is built on solana blockchain and is based on the versatile Anchor framework.
```
Comment (DiosAkhilleus) [inline solana/program_v2/packages/client/core/README.md:13]: "Solana" should be capitalized

File: solana/program_v2/packages/client/core/README.md [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+6. Supports multiple networks (devnet, testnet, mainnet-beta, etc) and multiple wallets (sollet, solflare, etc).
+7. On a Network and Gatekeeper level, the gateway protocol offers flexible payment configuration for Gateway Passes (e.g. pay for pass on issue, pay for pass on refresh, pay for pass on verify, etc).
+8. The Gateway protocol enables the usage of non-transferrable permissioned Gateway passes (mirroring properties of SBTs) that convey some property to the pass holder.
```
Comment (DiosAkhilleus) [inline solana/program_v2/packages/client/core/README.md:16]: Maintain consistent capitalization of Gateway Protocol... for instance in point 7 it says "gateway protocol" and here it says "Gateway protocol"

File: solana/program_v2/packages/client/core/README.md [hunk 4]
```diff
@@ -0,0 +1,537 @@
...
+
+1. AdminService provides create, update, close, and retrieve Network account methods.
+2. NetworkService provides create, update, close, retreive and setState of Gatekeeper methods.
+3. gatekeeperService provides issue, refresh, setState, setData, change the Associated Gatekeeper, expire, and verify of pass account methods.
+4. Typescript Client library for AdminService, NetworkService, and GatewayService.
+5. It is built on solana blockchain and is based on the versatile Anchor framework.
+6. Supports multiple networks (devnet, testnet, mainnet-beta, etc) and multiple wallets (sollet, solflare, etc).
+7. On a Network and Gatekeeper level, the gateway protocol offers flexible payment configuration for Gateway Passes (e.g. pay for pass on issue, pay for pass on refresh, pay for pass on verify, etc).
+8. The Gateway protocol enables the usage of non-transferrable permissioned Gateway passes (mirroring properties of SBTs) that convey some property to the pass holder. 
+9. Requirements and characteristics of a pass are defined within a Gatekeeper Network. Gatekeeper, belonging to Gatekeeper Networks are authoritative Issuers of Gateway Passes.
```
Comment (DiosAkhilleus) [inline solana/program_v2/packages/client/core/README.md:17]: Gatekeepers should be plural in the second sentence

File: solana/program_v2/packages/client/core/README.md [hunk 5]
```diff
@@ -0,0 +1,537 @@
...
+### Close a Network Account
+
+Close a network account. This will also close all gatekeepers associated with the network.
+
+```ts
+    // destination - account that receives the rent back.
+    await adminService.closeNetwork(destination: PublicKey, authority?: PublicKey).rpc();
+     
+```
+
```
Comment (DiosAkhilleus) [inline solana/program_v2/packages/client/core/README.md:250]: Maintain consistent spacing between sections

**Review by DiosAkhilleus — APPROVED** [review]

**Pull request merged**